tests/assertions/ocp4/ocp4-pci-dss-node-4.16.yml

```diff
@@ -176,7 +176,8 @@ rule_results:
   e2e-pci-dss-node-master-file-permissions-scheduler-kubeconfig:
     default_result: PASS
   e2e-pci-dss-node-master-file-permissions-var-log-kube-audit:
-    default_result: PASS
+    # nodes get INCONSISTENT because of https://bugzilla.redhat.com/show_bug.cgi?id=2001442
```

Comment:
Interesting - this is still present on newer releases even though the bug has been fixed?